tests/test_on_data/behavior/test_pose_estimation_interfaces.py

```diff
@@ -0,0 +1,565 @@
+
+python_version = version.parse(python_version())
+# TODO: remove after this is merged https://github.com/talmolab/sleap-io/pull/143 and released
+ndx_pose_version = version.parse(importlib_version("ndx-pose"))
+
+
+@pytest.mark.skipif(ndx_pose_version < version.parse("0.2.0"), reason="Interface requires ndx-pose version >= 0.2.0")
+class TestLightningPoseDataInterface(DataInterfaceTestMixin, TemporalAlignmentMixin):
+    data_interface_cls = LightningPoseDataInterface
+    interface_kwargs = dict(
+        file_path=str(BEHAVIOR_DATA_PATH / "lightningpose" / "outputs/2023-11-09/10-14-37/video_preds/test_vid.csv"),
+        original_video_file_path=str(
+            BEHAVIOR_DATA_PATH / "lightningpose" / "outputs/2023-11-09/10-14-37/video_preds/test_vid.mp4"
+        ),
+    )
+    conversion_options = dict(reference_frame="(0,0) corresponds to the top left corner of the video.")
+    save_directory = OUTPUT_PATH
+
+    @pytest.fixture(scope="class", autouse=True)
+    def setup_metadata(self, request):
+
+        cls = request.cls
+
+        cls.pose_estimation_name = "PoseEstimation"
+        cls.original_video_height = 406
+        cls.original_video_width = 396
+        cls.expected_keypoint_names = [
+            "paw1LH_top",
+            "paw2LF_top",
+            "paw3RF_top",
+            "paw4RH_top",
+            "tailBase_top",
+            "tailMid_top",
+            "nose_top",
+            "obs_top",
+            "paw1LH_bot",
+            "paw2LF_bot",
+            "paw3RF_bot",
+            "paw4RH_bot",
+            "tailBase_bot",
+            "tailMid_bot",
+            "nose_bot",
+            "obsHigh_bot",
+            "obsLow_bot",
+        ]
+        cls.expected_metadata = DeepDict(
+            PoseEstimation=dict(
+                name=cls.pose_estimation_name,
+                description="Contains the pose estimation series for each keypoint.",
+                scorer="heatmap_tracker",
+                source_software="LightningPose",
+                camera_name="CameraPoseEstimation",
+            )
+        )
+        cls.expected_metadata[cls.pose_estimation_name].update(
+            {
+                keypoint_name: dict(
+                    name=f"PoseEstimationSeries{keypoint_name}",
+                    description=f"The estimated position (x, y) of {keypoint_name} over time.",
+                )
+                for keypoint_name in cls.expected_keypoint_names
+            }
+        )
+
+        cls.test_data = pd.read_csv(cls.interface_kwargs["file_path"], header=[0, 1, 2])["heatmap_tracker"]
+
+    def check_extracted_metadata(self, metadata: dict):
+        assert metadata["NWBFile"]["session_start_time"] == datetime(2023, 11, 9, 10, 14, 37, 0)
+        assert self.pose_estimation_name in metadata["Behavior"]
+        assert metadata["Behavior"][self.pose_estimation_name] == self.expected_metadata[self.pose_estimation_name]
+
+    def check_read_nwb(self, nwbfile_path: str):
```

Comment:
We should add an explicit assertion for the skeletons in the `check_read_nwb()` functions for both DLC and Lightning.

Reply:
I added the DLC one, can you add the lightning one?

Reply:
Done!